dsl_survey/logql/overview.yaml

```diff
@@ -0,0 +1,112 @@
+title: Log Query Language
+
+abbreviation: LogQL
+
+copyright: Grafana Labs
+
+license: GNU Affero General Public License v3.0
+
+isOSS: true
+
+paradigms:
+  - functional
+  - declarative
+  - pipeline
+
+supportedTelemetryTypes:
+  - type: logs
+    callouts:
+      - title: Log Query
+        description: >
+          A query that returns individual log entries matching some criteria from the storage backend. All
+          log queries must specify a stream selector, and may include a subsequent log pipeline.
+
+          Example:
+            {stream selector} | log pipeline
```

Comment:
Descriptions can support markdown so you can wrap the code bits in ticks for an eventual UI.